sopel/module.py

```diff
@@ -472,6 +471,8 @@ def __call__(self, func):
 
         import sys
 
+        import sopel.test_tools
```

Comment:
Could you add a `# TODO: fix circular import with sopel.bot and sopel.test_tools`?

Reply:
:+1: Done.